### PR TITLE
070124

### PR DESCRIPTION
**Pull request by George-ast**

### CODE REVIEW

File: home7.1.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+search minimum value and print smth aka "'y' is minimum value'
+#where 'y' is variable name (not value)
+#advice use python debugger and different values to check your algorithm
+#optionally you can check your algorithm somehow with assert statements
+w, x, y, z = 100, 200, 40, 300
+"""
+
+w, x, y, z = 100, 200, 40, 300
+
+"""
```
Comment (antohaUa) [inline home7.1.py:14]: Use simple comment lines smth aka 
`# checking something blah blah blah`

File: home7.1.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Checking each condition whether the number is the smallest.
+"""
+
+if (x < w) and (x < y) and (x < z):
```
Comment (antohaUa) [inline home7.1.py]: Such set 
w, x, y, z = -1, -1, -1, 0  will brake your algorithm ))
Code should be updated

**Review by antohaUa — CHANGES_REQUESTED** [review]
Algorithm should be improved

File: home7.1.py (hunk 2 above)
Comment (George-ast) [reply home7.1.py]: что значит обновить код ?)

**Review by antohaUa — APPROVED** [review]
Fixes required for other variables too

**Pull request merged**